salt/core/systemd/init.sls

```diff
@@ -1,7 +1,19 @@
 # Stop RemoveIPC killing all processes by a user when they log out.
+{% if grains['osrelease']|float >= 20.04 %}
+/etc/systemd/logind.conf.d/customize.conf:
```

Comment:
We name the others `customization.conf`. Unless it's already taken, we should be consistent.

```suggestion
/etc/systemd/logind.conf.d/customization.conf:
```